src/DotVVM.Framework/Resources/Scripts/dotvvm-root.ts

```diff
@@ -124,9 +124,11 @@ const dotvvmExports = {
         logPostBackScriptError,
         level
     },
-    arrayHelper,
-    dictionaryHelper,
-    stringHelper
+    translations: {
+        arrayHelper,
```

Comment:
I think that when it's in the `translations` field, we could just call these `translations.array`, `translations.dictionary`, ...

Also, would be nice to add `as any` after this object, so it's not in the public typescript definitions, this is (mostly) not something users should use directly.